schemas.json

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
Because the file is used for tests I add to add it, also the Makefile command could be seen as one to update this file if the remote changed.

However I believe the file will stop changing at some point.